docs/source/guides/user/chapters/plugins.rst

```diff
@@ -175,10 +175,10 @@ test phases.  Here's a sequence of the job phases:
 Plugin writers can have their own code called at Avocado during a job
 by writing a that will be called at phase number 2 (``pre_tests``) by
```

Comment:
Not to be blamed by this commit, but this sentence seems to be broken.  The part "by writing a (???) that will be called" seems to be incomplete.